addons/website/views/snippets/s_text_cover.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_text_cover" name="Text cover">
+    <section class="s_text_cover">
+        <div class="container-fluid">
+            <div class="row flex-md-nowrap">
```

Comment:
I don't like to have exceptions per snippet: here the combination of `flex-md-nowrap` on the row + `flex-shrink-1` on the image col is what magically allows the column to apparently shrink instead of going to the next row when we increase the column on the left. It is also strange as reducing the first column size does not keep that behavior: it make the right column move to the left instead of growing.

I think this should be reserved to a more generic feature for all snippets @pierre-pde ?

Reply:
Hi @qsm-odoo , agreed, better to keep same behavior as other snippet..

Reply:
Thanks for the review. I'm sorry, here I didn't get if I should change something in this PR or not ? Is there already more generic way to achieve this result or not  ?

Reply:
"better to keep the same behavior as other snippets" => remove your specific classes and let this snippet behave like all others. If we want to improve that generic behavior, we will in a future task (not related to this snippet) :wink:

Reply:
Understood, thanks for the review again 👍